account-compression/sdk/.eslintrc.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     "no-unused-vars": "off",
     quotes: [
       "error",
-      "single",
+      "double",
```

Comment:
I know it's annoying, but I'd prefer that you do this in a separate PR since it's totally unrelated and adds a lot of churn to this PR

Reply:
Fixed!